### DISCUSSION TITLE
Kata 4

### DISCUSSION THREAD

**Discussion by MarioPiedras** [LaunchX-InnovaccionVirtual/CursoIntroPython#228] [Q&A]
![image](https://user-images.githubusercontent.com/99214519/153783749-0a96e27c-cc4e-4579-a453-7fd1e30a6e57.png)
Me podrian ayudar por favor la verdad se supone que me deberia rrojar otra cosa no?
![image](https://user-images.githubusercontent.com/99214519/153783775-2d48be80-d310-4ff5-b62b-cc1630c50982.png)

**Comment by alfredoaguiararce** [comment]
Podrías intentar ejecutando en orden las celdas, a veces en los notebooks, si en otra celda utilizamos una variable con un mismo nombre nos puede afectar nuestro flujo de trabajo.

Inténtalo volviendo a correr por orden las celdas para ver si eso soluciona tu error.

**Reply by MarioPiedras** [reply]
![image](https://user-images.githubusercontent.com/99214519/153786779-1aeea0af-93a3-4672-98e2-f5d342b8c2bc.png)
aprete ese boton y me hace lo mismo :(((

**Reply by JLuisPrz** [reply]
![153786779-1aeea0af-93a3-4672-98e2-f5d342b8c2bc](https://user-images.githubusercontent.com/83677705/153790092-43cede34-732e-42e9-8bb4-e8a14c99625f.png)
Hola.
En ese parte que te marque debe ser "key_word" y no "key_words".

**Reply by MarioPiedras** [reply]
Hola, me marca lo mismo :(((
![image](https://user-images.githubusercontent.com/99214519/153790601-8d13b073-9765-40bd-b397-ff7d776e9880.png)

**Reply by luisced** [reply]
<img width="1136" alt="solución" src="https://user-images.githubusercontent.com/63626850/153814663-17188aa1-e935-46dd-bf4a-0b24e42447a4.png">
Pues no sé si así era el ejercicio, pero esto es lo que dios me dio a entender :).

**Comment by miguelruizF** [comment]
Esto no es nada practico el hacer varios for. para eso esta el iterar sobre arrays, pero fue una de las maneras como realice el ejercicio. Lo unico que queda es cambiar el parametro en el metodo .find()

![image](https://user-images.githubusercontent.com/73854124/153791542-da481c69-4e59-4a27-b840-de611b314ab2.png)

**Comment by roy98123** [comment]
Hola! Además de los temas ya mencionados anteriormente como que tu "for" debe decir algo como: "for key_word in key_words", también tienes un error al momento de declarar esa variable, la estas declarando con paréntesis, debe declararse de la siguiente manera:

key_words = ["key1", "Key2","key3"]

Hago énfasis en que tienen que ser corchetes.  Si necesitas más ayuda consultando a través de una llamada compartiendo pantalla o algo puedes mandarme mensaje a este perfil por correo o a launchx03953@innovaccion.mx por medio de teams. 

![image](https://user-images.githubusercontent.com/55171384/153792641-232416ca-1be3-4266-af19-21674366ea05.png)

**Comment by EduardoOlvera1** [comment]
Hola amigo, yo hize el codigo asi, espero puedas darte una idea.

igual en la variable sentence deberias tener las oraciones en una **lista**, con .split(".")
vuelve a checarla o te recomiendo imprimir sentence en consola para que cheques si realmente tienes las oraciones

![imagen](https://user-images.githubusercontent.com/99152643/153793659-1f24a7b8-688b-4ff1-87d0-76b7ad826706.png)

**Comment by MarioPiedras** [comment]
Gracias a todos por su ayuda lo intentare resolverlo ahora estoy probando sus soluciones, soy nuevo en esto todavia no entiendo con exactitud que hago :(((

**Reply by miguelruizF** [reply]
Te recomiendo que cheques devdocs.io
Viene documentacion de diversos tipos de lenguajes, a mi me ayudo para aprender mas sobre JavaScript y PHP
Espero te sirva
![image](https://user-images.githubusercontent.com/73854124/153799386-b0c04991-1974-44e6-9ade-ae39b58d2a8a.png)
